.pre-commit-config.yaml

```diff
@@ -0,0 +1,42 @@
+# Apply to all files without committing:
+#   pre-commit run --all-files
+# Update this file:
+#   pre-commit autoupdate
+default_language_version:
+  # Use latest LTS release
+  node: 22.14.0
+repos:
+  - repo: meta
```

Comment:
Where's this repo URL?

Reply:
The `meta` repo is bundled by default with `pre-commit`.